mountpoint-s3/Cargo.toml

```diff
@@ -56,6 +56,7 @@ fuse_tests = []
 s3_tests = []
 shuttle = []
 put = []
+checksum = []
```

Comment:
Could you add a comment for this feature? It's a temporary feature flag to enable read-path checksums while we iron out performance. (We should have done the same for `put`).